src/templates/script/getScriptVerifiedTemplateIds.ts

```diff
@@ -10,7 +10,9 @@ export function getScriptVerifiedTemplateIds(version: string): (string | RegExp)
         'BlobTrigger',
         'HttpTrigger',
         'QueueTrigger',
-        'TimerTrigger'
+        'TimerTrigger',
+        'SQLInputBinding',
+        'SQLOutputBinding'
```

Comment:
Nit, for one less line-change in the future:
```suggestion
        'SQLOutputBinding',
```